src/core_atmosphere/Makefile

```diff
@@ -28,23 +30,30 @@ post_build:
 	cp default_inputs/* $(ROOT_DIR)/default_inputs/.
 	( cd $(ROOT_DIR)/default_inputs; for FILE in `ls -1`; do if [ ! -e ../$$FILE ]; then cp $$FILE ../.; fi; done )
 
+physcore_init:
+	(cd physics;  $(MAKE) core_physics_init )
+
```

Comment:
It appears to me that `phscore_init` can be embedded in `chemcore`.  The hierarchy for `physcore_init` may be different from the rest:`chemcore physcore dycore diagcore atmcore utilities` in the Makefile.

Reply:
For clarity, I do not want to embed physics in chemistry.

Reply:
Since this make segment is added for the OPP code, I assumed it is needed for OPP not for physics_wrf, as it is extracted from the latter.
Initialization literally should be logically included in core_init_atmosphere.  
Is this the way how wrf-chem is included in WRF?
Is this name physcore_init intended for nondycore_init, and are there other ensuing usage for it so that it has to be parallel with physics and chemistry build in the makefile.

Reply:
I agree with @metdyn's comment, mostly because I am confused about the motivation for creating this new "make" mode (sorry I do not know the correct terminology).  In short, the make dependencies between the make modes look overly complex compared to the `release-stable` branch.  It seems like the chem make process could be made independent from the physics make procedure.

Although at this point I care most about whether we are merging code that is consistent with what will be merged to the upstream `MPAS-Dev` fork.  Have these changes already been merged upstream to `MPAS-Dev` as-is?

Reply:
I think the new `physcore_init` target is used to build files in the `physics/physics_wrf` directory that are needed by `chemistry`; at present, these files would only include `module_ra_cam_support.F` file. Would it help if this build target were to be renamed something like `physics_shared` to indicate that it leads to the compilation of any physics files that are shared with other model components (e.g., chemistry)?